### DISCUSSION TITLE
When I open index.html in my browser, it is blank? Should I have Linux installed?

### DISCUSSION THREAD

**Discussion by TheRealDigitalMaster** [google/model-viewer#3929] [Q&A]
### Description

When I open the editor page, it shows this:
![image](https://user-images.githubusercontent.com/112659665/197556579-1cf5cee7-646c-4ed6-a8c9-4b2958735983.png)
What it should do is open the editor. And when I go through the files and finally find the correct editor, it's blank:
![image](https://user-images.githubusercontent.com/112659665/197557575-6620cf37-90e1-4123-8969-60039aa15d8c.png)
Please fix this. Should I have Linux installed?

### Version

- model-viewer: v2.0.2

### Browser Affected

- [x] Chrome, version: 106.0.5249.112

### OS

- [x] Chrome OS

### AR

I don't have AR I only have Chrome browser on Chromebook. Can you guys make this compatible for people with/without Linux?

**Comment by elalish** [comment]
The easy way to open the editor is to navigate to the public version: https://modelviewer.dev/editor/

The only reason to serve it locally (that I can think of) is if you're developing it. In that case you'll want to start by building locally.

**Comment by TheRealDigitalMaster** [comment]
This website is blocked on my account (because of Family Link), so I am looking for other things to download and use as an alternative. Like aframe, I downloaded it from GitHub, but I can't figure out how to import gltf/glb files. So then I was looking for model viewers, like [https://modelviewer.dev/editor](https://modelviewer.dev/editor)

**Comment by TheRealDigitalMaster** [comment]
But at the beginning of my 3d designing, I used [Voxel Builder](https://github.com/nimadez/voxel-builder) (also downloaded from GitHub).

**Comment by elalish** [comment]
Weird, why would Family Link block modelviewer.dev? Anyway, you can follow our README to get a build environment set up, but I have the feeling the much easier thing would be to ask your parents to unblock modelviewer.dev. It's just a Google open-source documentation site.

**Comment by TheRealDigitalMaster** [comment]
Ok :) I will ask them. I am making a(n) Hawkins Lab upside down portal room with Demogorgon (Stranger Things setup) in aFrame. So I am looking to view my models in modelviewer.dev to then import it into my 3d vr world.
